### PR TITLE
feat(TNLT-4855): Hide save star on tablet

### PR DESCRIPTION
**Pull request by c-j-j**
https://nidigitalsolutions.jira.com/browse/TNLT-4855

### CODE REVIEW

**Comment by aronshamash** [comment]
@c-j-j no snap updates ?

**Comment by c-j-j** [comment]
@aronshamash no - there are currently no unit tests for the PositionedTileStar itself, and it is mocked everywhere else. Happy to add unit tests for the star if it'll help - what do you think?

**Comment by aronshamash** [comment]
> @aronshamash no - there are currently no unit tests for the PositionedTileStar itself, and it is mocked everywhere else. Happy to add unit tests for the star if it'll help - what do you think?

Lets leave as is for now.

**Review by aronshamash — APPROVED** [review]

File: packages/edition-slices/src/tiles/shared/positioned-tile-star.js [hunk 1]
```diff
@@ -11,14 +11,17 @@ import {
...
+import { ResponsiveContext } from "@times-components-native/responsive";
 
 const PositionedTileStar = ({
   centeredStar = false,
   underneathTextStar = false,
   customStyles = {},
   ...props
 }) => {
-  if (!isSaveSupported) return null;
+  const { isTablet } = useContext(ResponsiveContext);
```
Comment (Geoff-Ford) [inline packages/edition-slices/src/tiles/shared/positioned-tile-star.js:22]: Should we follow that pattern of creating a `useResponsiveContext` method to `@times-components-native/responsive`? What do you think?

**Review by Geoff-Ford — APPROVED** [review]

File: packages/edition-slices/src/tiles/shared/positioned-tile-star.js (hunk 1 above)
Comment (c-j-j) [reply packages/edition-slices/src/tiles/shared/positioned-tile-star.js:22]: Sounds good - I can do that in another PR - and then update all places that have `useContext(ResponsiveContext)`?

**Pull request merged**